External/3rdParty/Scintilla/additional_keywords.patch

```diff
@@ -1,65 +1,65 @@
-diff -r 1092d3d24959 include/SciLexer.h
---- a/include/SciLexer.h	Tue Jan 15 09:51:15 2013 +1100
-+++ b/include/SciLexer.h	Wed Feb 06 22:13:27 2013 +0100
-@@ -161,6 +161,9 @@
- #define SCE_C_TRIPLEVERBATIM 21
- #define SCE_C_HASHQUOTEDSTRING 22
- #define SCE_C_PREPROCESSORCOMMENT 23
-+#define SCE_C_WORD3 24
-+#define SCE_C_WORD4 25
-+#define SCE_C_WORD5 26
+diff --git a/include/SciLexer.h b/include/SciLexer.h
+--- a/include/SciLexer.h
++++ b/include/SciLexer.h
+@@ -177,6 +177,9 @@
+ #define SCE_C_USERLITERAL 25
+ #define SCE_C_TASKMARKER 26
+ #define SCE_C_ESCAPESEQUENCE 27
++#define SCE_C_WORD3 100
```

Comment:
I started these at 100 to hopefully avoid conflicts in the future so patches can be applied more easily. Also helps distuingish custom and "real" defines.